CONTRIBUTING.md

```diff
@@ -6,6 +6,10 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+For technical issues related to using Node.js that are not issues with the
```

Comment:
I think you're probably right. I may have tried to go a bit to dry and
technical with it. Should I do a new commit?

On Tue, Oct 6, 2015 at 11:00 PM Rich Trott <notifications@github.com> wrote:

> In CONTRIBUTING.md
> <https://github.com/nodejs/node/pull/3233#discussion_r41354279>:
>
> > @@ -6,6 +6,10 @@ When opening new issues or commenting on existing issues on this repository
> >  please make sure discussions are related to concrete technical issues with the
> >  Node.js software.
> >
> > +For technical issues related to using Node.js that are not issues with the
>
> What do you think of something more like this: 'For general help using
> Node.js, please file an issue at...` I think that may be easier to grok
> than "issues related to using Node.js that are not issues with the Node.js
> software itself"...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/3233/files#r41354279>.
>
-- 
Best Regards,

Doug Shamoo